guides/code-style/lints.rake

```diff
@@ -1,8 +1,17 @@
 require 'rubocop/rake_task'
 
-RuboCop::RakeTask.new
+task :jshint_custom do
+  puts 'Running JS lints...'
+  if system("bundle exec rake jshint:lint['.jshint.yml']")
```

Comment:
With SCSS lint there was a particular reason. IIRC, the task definition didn't take config into account or something like that. Maybe JSHint task is better and we can directly call it?